clients/src/main/java/org/apache/kafka/clients/tools/VerifiableProducer.java

```diff
@@ -124,6 +132,23 @@ private static ArgumentParser argParser() {
                 .choices(0, 1, -1)
                 .metavar("ACKS")
                 .help("Acks required on each produced message. See Kafka docs on request.required.acks for details.");
+        
+        parser.addArgument("--producer-prop")
+                .action(Arguments.append())
+                .required(false)
+                .type(String.class)
+                .dest("userSpecifiedProducerProps")
+                .metavar("PRODUCER-PROP")
+                .help("Any custom producer properties of the form key=val.");
+
+        parser.addArgument("--close-timeout")
+                .action(store())
+                .required(false)
+                .setDefault(10L)
```

Comment:
Yeah I'm not sure the best solution here.

Basically every replication test results in waiting for a very long time after the initial trigger of the close() method, and typically the wait can be *much* longer than all of the other test logic.

One alternative method might be the throttle the throughput more aggressively.